cmake/messageFunc.cmake

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
Could you please add a one-line comment showing an example of call ?
Usage is not that obvious